k8s/trouble-shooting.md

```diff
@@ -0,0 +1,241 @@
+# Troubleshooting
```

Comment:
For consistency, could you please use `troubleshooting` (one word) for the name of this file & in the line below?

Reply:
Yes